src/platforms/common/performance/index.mdx

```diff
@@ -94,11 +94,11 @@ Test out tracing by starting and finishing a transaction, which you _must_ do so
 
 While you're testing, set <PlatformIdentifier name="traces-sample-rate" /> to `1.0`, as that ensures that every transaction will be sent to Sentry.
```

Comment:
I have a suggestion here - how about we introduce what sampling does generically first and then go into recommended settings for pre-prod vs. prod environments? 

"Sampling enables you to collect representative data without overwhelming your system or your [transaction quota] (/product/accounts/quotas/). While you're testing, you're likely doing so in a pre-production environment with limited traffic, so we recommend setting <PlatformIdentifier name="traces-sample-rate" /> to `1.0`, as that ensures that every transaction will be sent to Sentry.

Once testing is complete, you may consider changing your sampling configuration when rolling out to production if overhead is a concern. If you have a high-load, backend application, you may want to consider setting a lower <PlatformIdentifier name="traces-sample-rate" /> value, or switching to using <PlatformIdentifier name="traces-sampler" /> to selectively sample and filter your transactions, based on contextual data."